src/aws/flb_aws_credentials_http.c

```diff
@@ -29,11 +29,10 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
-/* HTTP Credentials Endpoints have a standard set of JSON Keys */
-#define AWS_HTTP_RESPONSE_ACCESS_KEY   "AccessKeyId"
-#define AWS_HTTP_RESPONSE_SECRET_KEY   "SecretAccessKey"
-#define AWS_HTTP_RESPONSE_TOKEN        "Token"
-#define AWS_HTTP_RESPONSE_EXPIRATION   "Expiration"
+#define AWS_CREDENTIAL_RESPONSE_ACCESS_KEY   "AccessKeyId"
+#define AWS_CREDENTIAL_RESPONSE_SECRET_KEY   "SecretAccessKey"
+#define AWS_HTTP_RESPONSE_TOKEN              "Token"
```

Comment:
Based on the docs, credential process can return a key `SessionToken`

Reply:
Oh I see you solved this a new argument to the parsing function. That works. I probably would've just added both constants and then had the function check for both of them.